spec/poros/image_options_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.describe "Image Options" do
+  it "exists" do
+    json = {:data=>
+                  [{:id=>"96eec9e5-09ee-4c8a-bc1c-22960e8d39b9",
+                    :type=>"image",
+                    :attributes=>
+                     {:id=>"96eec9e5-09ee-4c8a-bc1c-22960e8d39b9",
+                      :url=>
+                       "https://api.deepai.org/job-view-file/96eec9e5-09ee-4c8a-bc1c-22960e8d39b9/outputs/output.jpg",
+                      :source=>"Deep AI"}},
+                   {:id=>"2083492",
+                    :type=>"image",
+                    :attributes=>
+                     {:id=>2083492,
+                      :url=>
+                       "https://pixabay.com/get/54e0dd404e5bae14f1dc846096293f7c123cdeed574c704f752f7bd69145c059_640.jpg",
+                      :source=>"Pixabay"}}]}
```

Comment:
Or is it Pics O' Bey? ![](https://media4.giphy.com/media/6J2kRTRUht18A/giphy.gif)